### DISCUSSION TITLE
Inbox improvements / Inbox search

### DISCUSSION THREAD

**Discussion by aaronzorzi** [AprilSylph/XKit-Rewritten#386] [Suggestion Box]
Hello! Would it be possible at any point to add some improvements for the inbox? I run some very active blogs that get thousands of messages, and its an absolute nightmare to try and handle all of them with tumblr's native inbox. Will XKit Rewritten ever be adding inbox improvements of any kind, or an inbox searching feature? Thanks for all the work you do!

**Comment by AprilSylph** [comment]
What inbox improvements are you thinking of? Quick Tags already has options for automatically tagging asks you answer.

Inbox searching is not feasible, and even if it were, it would just be a shortcut to loading every inbox item and running a CTRL+F text search on them. Stay tuned for "Scroll to Bottom" to make a return to help with this, though.
